tools/run-tests.py

```diff
@@ -246,13 +246,15 @@ def create_binary(job, options):
     if binary_key in BINARY_CACHE:
         ret, build_dir_path = BINARY_CACHE[binary_key]
         sys.stderr.write(f'(skipping: already built at {build_dir_path} with returncode {ret})\n')
+        sys.stderr.flush()
         return ret, build_dir_path
 
     try:
         subprocess.check_output(build_cmd)
         ret = 0
     except subprocess.CalledProcessError as err:
-        print(err.output.decode("utf8"))
+        # For python <-> native program, we use default encoding with error='ignore' to not lost data
+        print(err.output.decode(errors="ignore"))
```

Comment:
For win32, the `native program` will output with `GBK` encoding,  the build command is native program, maybe a better option is use settings.native_program_encoding ?

Reply:
Oh, I gotcha it, it's because when I create this MR is for support `python2`, if `python 2` support is dropped, then I can use `errors='ignore'` as `open` parameter

Reply:
I am still confused about the `errors="ignore"` parameter. IMO, this can lead to data loss.

> 'ignore' ignores errors. Note that ignoring encoding errors can lead to data loss.

source: https://docs.python.org/3/library/functions.html#open

My concern is that I don't know what happens with those test files which contains invalid unicode characters intentionally to test the parser.

For example:
- https://github.com/jerryscript-project/jerryscript/issues/1549
- https://github.com/jerryscript-project/jerryscript/blob/dfa9afbf6eec12c5f9607e86be681afbef806cb5/tests/jerry/fail/regression-test-issue-1549.js

Reply:
This is not a issue, it's
```
PS C:\work\study\languages\typescript\jerryscript> C:\work\study\languages\typescript\jerryscript\build\tests\jerry_tests-external_context\local\bin\jerry.exe tests/jerry/fail/regression-test-issue-1549.js
Unhandled exception: SyntaxError: Input is not a valid UTF-8 encoded string.
```

The main fixes for windows is for avoid test failure when the test file output 'utf8' characers that can not be recognized by 'gbk' on windows,
there is no decoding error with 'utf8' on linux

Reply:
anyway, your concern addressed, only ignore on win32

Reply:
Oh, I re-test it locally, found  I've already strip the cmd.exe prefix, so there is no need handling win32 case anymore
```
 def get_python_cmd_prefix():
     # python script doesn't have execute permission on github actions windows runner
-    return get_platform_cmd_prefix() + [sys.executable or 'python']
+    return [sys.executable or 'python']
```
So I revert this changes